crates/universal-sierra-compiler-api/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "universal-sierra-compiler-api"
+version = "1.0.0"
+edition.workspace = true
+
+[dependencies]
+anyhow.workspace = true
+assert_fs.workspace = true
+camino.workspace = true
+indoc.workspace = true
+scarb-metadata.workspace = true
+scarb-ui.workspace = true
+serde.workspace = true
+serde_json.workspace = true
+thiserror.workspace = true
+which.workspace = true
+semver.workspace = true
+tempfile.workspace = true
```

Comment:
Please make sure to remove unused dependencies (I don't see a use for `indoc`, `semver`, `thiserror`, `camino`, `scarb-ui` etc.)